server/handlers/transferHandler/schemas.js

```diff
@@ -12,16 +12,15 @@ const transferPostSchema = Joi.alternatives()
         tokens: Joi.array().items(Joi.string()).required().unique(),
         sender_wallet: Joi.alternatives().try(Joi.string()).required(),
         receiver_wallet: Joi.alternatives().try(Joi.string()).required(),
-        // TODO: add boolean for claim, but default to false.
-        claim: Joi.boolean(),
+        claim: Joi.boolean().default(false),
       }),
       otherwise: Joi.object({
         bundle: Joi.object({
           bundle_size: Joi.number().min(1).max(10000).integer(),
         }).required(),
         sender_wallet: Joi.string().required(),
         receiver_wallet: Joi.string().required(),
-        claim: Joi.boolean().required(),
+        claim: Joi.boolean().default(false),
```

Comment:
Don't think this default will work, due to the way we are calling the validations with validateAsync. Can you confirm this?

Reply:
You are right. The claim value would just be left undefined in this case. I've fixed it.